src/__test__/Results.test.tsx

```diff
@@ -2,30 +2,127 @@ import Results from "../components/Results";
 import React from "react";
 import ReactDOM from "react-dom";
 import { render, cleanup } from "@testing-library/react";
+import { CATEGORY_NAMES } from "../constants";
+
+const defaultAllCorrectAnswers = {
```

Comment:
We need to add `freeCodeCampCorrect: 0,` because that category is listed in the CATEGORY_NAMES constant